Formula/trojan-go.rb

```diff
@@ -0,0 +1,105 @@
+class TrojanGo < Formula
+  desc "Trojan proxy in Go"
+  homepage "https://p4gefau1t.github.io/trojan-go/"
+  url "https://github.com/p4gefau1t/trojan-go.git",
+      tag: "v0.10.0",
+      revision: "1d9c46760efe143aa1f9bd7e2f7cf460cb364259"
```

Comment:
```suggestion
      revision: "a9ce6d4ada9143d58e8d1373d23f9ebfe871119e"
```

Reply:
```
$ brew install --build-from-source trojan-go
Error: v0.10.0 tag should be a9ce6d4ada9143d58e8d1373d23f9ebfe871119e
but is actually 1d9c46760efe143aa1f9bd7e2f7cf460cb364259
```

Reply:
No need to guess

![image](https://user-images.githubusercontent.com/10487845/117533349-e1e01a80-b01e-11eb-96a0-5ac9e65abeef.png)

Reply:
I understand that, I will try cleaning my cache.

Reply:
```
$ trojan-go -version
Trojan-Go v0.10.0
Go Version: go1.16.3
OS/Arch: darwin/amd64
Git Commit: a9ce6d4ada9143d58e8d1373d23f9ebfe871119e

Developed by PageFault (p4gefau1t)
Licensed under GNU General Public License version 3
GitHub Repository:	https://github.com/p4gefau1t/trojan-go
Trojan-Go Documents:	https://p4gefau1t.github.io/trojan-go/
```
well done